src/getCoinCombination.test.js

```diff
@@ -7,4 +7,28 @@ describe('getCoinCombination', () => {
     expect(getCoinCombination)
       .toBeInstanceOf(Function);
   });
+
+  it(`order from big sum`, () => {
+    const result = getCoinCombination(250);
+
+    expect(result).toEqual([0, 0, 0, 10]);
+  });
+
+  it(`order from small sum`, () => {
+    const result = getCoinCombination(3);
+
+    expect(result).toEqual([3, 0, 0, 0]);
+  });
+
+  it(`order < dollar sum`, () => {
+    const result = getCoinCombination(79);
+
+    expect(result).toEqual([4, 0, 0, 3]);
+  });
+
+  it(`is Array`, () => {
+    const result = getCoinCombination();
+
+    expect(Array.isArray(result));
```

Comment:
The test case `is Array` checks if the result is an array, but the `expect` statement is incomplete. It should be `expect(Array.isArray(result)).toBe(true);` to properly assert that the result is indeed an array.